Makefile

```diff
@@ -355,10 +355,5 @@ PKG_MAN_OPTS ?= "$(PKG_CHANNELS) $(PKG_DEFAULT_CHANNEL)"
 
 # Build the packagemanifests
 .PHONY: release/generate-bundle
-release/generate-bundle: dev/generate
+release/generate-bundle:
 	bazel run //hack:bundle -- $(RH_BUNDLE_VERSION) $(RH_OPERATOR_IMAGE) $(PKG_MAN_OPTS) $(RH_COCKROACH_DATABASE_IMAGE)
-
-#  Build the OPM bundle
-.PHONY: release/opm-build-bundle
-release/opm-build-bundle: release/generate-bundle
```

Comment:
This wasn't being used anymore, so I just 🔥'd it

Reply:
👋